java/com/epam/ta/salad/Repository/SQLService/SaladServiceImpl.java

```diff
@@ -1,4 +1,51 @@
 package com.epam.ta.salad.Repository.SQLService;
 
-public class SaladServiceImpl {
-}
+import Entities.Salad;
+import Entities.Vegetable;
+import com.epam.ta.parking.util.NullCheckUtil;
+import com.epam.ta.salad.Repository.Repository.SaladRepository;
+import com.epam.ta.salad.Repository.RepositorySpecification.VegetableSpecification;
+import com.epam.ta.salad.Repository.SQLRepository.VegetableRepositoryImpl;
+import com.epam.ta.salad.Repository.Service.SaladService;
+import exceptions.NullVegetableException;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class SaladServiceImpl implements SaladService {
+    @Override
+    public Salad add(Salad salad) throws SQLException, ClassNotFoundException {
+        return null;
```

Comment:
what is it? why do you return null?

Reply:
> what is it? why do you return null?

because as I see, I inherit from SaladService which inherited From BaseService. I suppose, we don't need to add any logic to the Salad such as add or update, because we do that with vegetables. So, do we have another way how i can drop non-used metods?